_includes/desktop-license-update.md

```diff
@@ -5,6 +5,6 @@
     - Our [Docker Subscription Service Agreement](https://www.docker.com/legal/docker-subscription-service-agreement){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} includes a change to the terms of use for Docker Desktop
     - It **remains free** for small businesses (fewer than 250 employees AND less than $10 million in annual revenue), personal use, education, and non-commercial open source projects.
     - It requires a paid subscription for professional use in larger enterprises.
-    - The effective date of these terms is August 31, 2021. There is a **grace period** until January 31, 2022 for those that will require a paid subscription to use Docker Desktop.
+    - The effective date of these terms is August 31, 2021. A **grace period** ran until January 31, 2022.
```

Comment:
Would it make sense to remove the statement about the grace period as it has already ended?

Reply:
That was my instinct too, but I erred on the side of caution. I pushed a new commit removing reference to it entirely in most cases.